src/backend/AbsSecBlock.c

```diff
@@ -312,14 +312,15 @@ int BSL_AbsSecBlock_EncodeToCBOR(const BSL_AbsSecBlock_t *self, BSL_Data_t alloc
         QCBOREncode_CloseArray(&encoder);
     }
 
-    UsefulBufC output_buf;
-    QCBORError qcbor_err = QCBOREncode_Finish(&encoder, &output_buf);
-    if (qcbor_err != QCBOR_SUCCESS)
+    size_t     encode_sz;
+    QCBORError qcbor_err = QCBOREncode_FinishGetSize(&encoder, &encode_sz);
```

Comment:
As a workaround to #35 go back to using `QCBOREncode_Finish()` as was there, but accepting the `QCBOR_ERR_BUFFER_TOO_SMALL` result as valid.